tests/dapscompare-tests/db5-procedure/xml/test.xml

```diff
@@ -49,27 +49,195 @@
  </info>
 
  <chapter xml:id="cha.procedure">
-  <title>Procedures</title>
```

Comment:
Hm, did I accidentally put the test case stuff into the same commit here on Friday? If so, that was not intended...